src/plugins/oauth2/flow/passwordFlow.ts

```diff
@@ -44,6 +44,8 @@ class PasswordFlow implements OpenIdFlow {
             tokenEndpoint: config.tokenEndpoint,
             grantType: 'password',
             username: config.username,
+            audience: config.audience,
```

Comment:
Was the change intentional, or should this not have been on line 33 in the body?

Reply:
Well, I don't think that the audience parameter is explicitly forbidden for the ROPC flow, so it should be optional for all flows.

Reply:
From RFC8693:
> OPTIONAL. The logical name of the target service where the client intends to use the requested security token. This serves a purpose similar to the resource parameter but with the client providing a logical name for the target service. Interpretation of the name requires that the value be something that both the client and the authorization server understand. An OAuth client identifier, a SAML entity identifier [OASIS.saml-core-2.0-os], and an OpenID Connect Issuer Identifier [OpenID.Core] are examples of things that might be used as audience parameter values. However, audience values used with a given authorization server must be unique within that server to ensure that they are properly interpreted as the intended type of value. Multiple audience parameters may be used to indicate that the issued token is intended to be used at the multiple audiences listed. The audience and resource parameters may be used together to indicate multiple target services with a mix of logical names and resource URIs.

Reply:
But it is not added to the flow. Only to the session object.

Reply:
Ah, true, I'll add that tomorrow

Reply:
@AnWeber fixed